Runtime/Scripts/IDCCLauncher.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Unity.MeshSync
+{
+    public enum RunMode
```

Comment:
Please make this into an internal class, unless we want to open this API to users.